### PR TITLE
OCM-10614 | feat: allow to verify error message for subsystem tests

### PR DESCRIPTION
**Pull request by gdbranco**
[OCM-10614](https://issues.redhat.com//browse/OCM-10614)
**Change type**
- [ ] New feature
- [x] Bug fix
- [ ] Build
- [ ] CI
- [ ] Documentation
- [ ] Performance
- [x] Refactor
- [x] Style
- [ ] Unit tests
- [x] Subsystem tests

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

**Comment by marcolan018** [comment]
is it useful to add function `func (r *TerraformRunner) Output() RunOutput` to test the `terraform output`?

**Comment by gdbranco** [comment]
> is it useful to add function `func (r *TerraformRunner) Output() RunOutput` to test the `terraform output`?

Yeah, nothing uses it atm so I didn't include. But sure

File: subsystem/framework/framework.go [hunk 1]
```diff
@@ -243,32 +211,30 @@ func (r *TerraformRunner) Run(args ...string) int {
 	GinkgoWriter.Println(&outb)
 	GinkgoWriter.Println(&errb)
 
-	r.LastRunOutput = RunOutput{
-		out: outb.String(),
-		err: errb.String(),
+	return RunOutput{
+		out:        outb.String(),
+		err:        errb.String(),
+		StatusCode: cmd.ProcessState.ExitCode(),
```
Comment (marcolan018) [inline subsystem/framework/framework.go:217]: a np one
can we change the attribute name `StatusCode` to `ExitCode`? Current one easily to mislead others to regard it as the status code from http response

Comment (gdbranco) [reply subsystem/framework/framework.go:217]: Oh I forgot to do that

**Pull request merged**